guardrails/guard.py

```diff
@@ -1259,6 +1261,19 @@ def to_dict(self) -> Dict[str, Any]:
 
         return i_guard_dict
 
+    def augment_tools_with_schema(
```

Comment:
Not a blocker bc we can always deprecate in favor of something else once we make a decision, but since this is intended to facility function calling can we make the name reflect that?

Reply:
updated to add_json_function_calling_tool